master/README.md

```diff
@@ -0,0 +1,45 @@
+# Structure of Master
+
+## Structure
+
+The master module consists of several components:
+
+- Master Server.
+  - provide grpc/http api, including
+    - operate cluster
+      - register/delete server
+      - put heartbeat (keep alive)
+    - user interface
+      - submit job
+      - update job
+      - delete job
+  - ExecutorManager
+    - Handle Heartbeat
+    - Maintain the stats of Executors
+      - Notify Resource Manager Udpate the status infos
+    - Once an executor is offline, it should notify the resource manager to reschedule all the tasks on it.
+- ResourceManager
+  - ResourceManager maintains the status of all executors and tasks.
+  - It implements several interfaces, including:
+    - GetResourceShapshot
+    - ApplyNewTasks
+    - UpdateExecutorStats
+    - Register/UnRegister Executor
```

Comment:
Seems in current implementation, executor registers itself to master, and master appends executor in `ExecutorManager`.

Reply:
yeah, the master exposed this interface to executor rather than the users. this doc is confusing, I will change it